generic/tests/build.py

```diff
@@ -9,8 +9,8 @@ def run(test, params, env):
     :param params: Dictionary with test parameters.
     :param env: Test environment.
     """
-    srcdir = params.get("srcdir", test.srcdir)
-    params["srcdir"] = srcdir
+    workdir = params.get("workdir", test.workdir)
```

Comment:
Suggest to keep the original param name, or you have to check whether the configuration files  should be updated correspondingly.

Reply:
I have check it should be no srcdir in repos any more after this patch is merged.

Reply:
ok, thanks.